examples/all_widgets.py

```diff
@@ -43,6 +43,9 @@
 
 # waffle = Waffle(app)
 
+# labelBox = LabelBox(app, "This is a label")
```

Comment:
Is there a reason this is commented out?

Reply:
All the other widgets are commented out in this example file so I just did the same.